include/swift/Subsystems.h

```diff
@@ -186,7 +186,7 @@ namespace swift {
 
   /// Once type checking is complete, this walks protocol requirements
   /// to resolve default witnesses.
-  void finishTypeChecking(SourceFile &SF);
+  void finishTypeCheckingOfFile(SourceFile &SF);
```

Comment:
Whoops, would still prefer `finishTypeCheckingFile` for this.

Reply:
Done.